src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNet/AspNetMvcIntegration.cs

```diff
@@ -80,8 +80,8 @@ public static Scope CreateScope(object controllerContext)
 
                         if (route != null)
                         {
-                            var resourceUrl = route.Url.ToLowerInvariant();
-                            if (resourceUrl.First() != '/')
+                            var resourceUrl = (route.Url ?? string.Empty).ToLowerInvariant();
```

Comment:
I think this is a little easier to read (and avoids calling `ToLowerInvariant()` on an empty string in the null case):
```suggestion
                            var resourceUrl = route.Url?.ToLowerInvariant() ?? string.Empty;
```